templates/default/sql/dml/undo/2.6.0__undo.sql.erb

```diff
@@ -1,4 +1,7 @@
 DELETE FROM `hopsworks`.`variables` WHERE `id`='fs_py_job_util';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='fs_java_job_util';
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("fs_job_util", "hdfs:///user/<%= node['hadoop_spark']['user'] %>/hsfs_utils-<%= node['hadoop_spark']['hsfs']['utils']['version'] %>.py");
-DELETE FROM `hopsworks`.`variables` WHERE `id`='kube_skip_namespace_creation';
\ No newline at end of file
+DELETE FROM `hopsworks`.`variables` WHERE `id`='kube_skip_namespace_creation';
+
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("kube_kfserving_installed", "<%= node['kube-hops']['kfserving']['enabled'] %>", 1);
```

Comment:
Hmm I think actually this should also be ['kube-hops']['kserve']['enabled'] instead of as ['kube-hops']['kfserving']['enabled'] will not exist in the cookbook version that runs this undo.

Reply:
Fixed